packages/gasket-plugin-jest/lib/index.js

```diff
@@ -18,8 +18,8 @@ module.exports = {
 
         if (isReactProject) {
           files.add(
-            path.join(__dirname, '..', 'generator', '*'),
-            path.join(__dirname, '..', 'generator', '**', '*')
+            `${generatorDir}/*`,
```

Comment:
Will these file paths work in a path-separator-neutral manner?

Reply:
Yea this uses `glob` under the hood, which wants forward-slashes only ([docs](https://github.com/isaacs/node-glob#windows))